appveyor.yml

```diff
@@ -49,6 +49,9 @@ before_test:
       ($cert = Import-PfxCertificate -FilePath resources/cert.pfx -CertStoreLocation cert:\\LocalMachine\\My -Password (ConvertTo-SecureString -String bolt -Force -AsPlainText)) | Format-List
       New-WSManInstance -ResourceURI winrm/config/Listener -SelectorSet @{Address='*';Transport='HTTPS'} -ValueSet @{Hostname='localhost';CertificateThumbprint=$cert.Thumbprint} | Format-List
 
+      docker run -d -p 35985:5985 microsoft/windowsservercore
```

Comment:
Networking with docker-compose on Windows was unreliable, but we don't need it. So created two individual instances.

Reply:
Do you have a windows test environment with bolt and docker configured to test this out? If so did you do that on a vmpooler instance?